agentlocal/agent_local.go

```diff
@@ -135,10 +136,11 @@ func (s *Server) Status(ctx context.Context, req *agentlocalpb.StatusRequest) (*
 	var serverInfo *agentlocalpb.ServerInfo
 	if u := s.cfg.Server.URL(); u != nil {
 		serverInfo = &agentlocalpb.ServerInfo{
-			Url:         u.String(),
-			InsecureTls: s.cfg.Server.InsecureTLS,
-			Connected:   connected,
-			Version:     md.ServerVersion,
+			Url:          u.String(),
+			InsecureTls:  s.cfg.Server.InsecureTLS,
+			Connected:    connected,
+			Version:      md.ServerVersion,
+			AgentVersion: version.Version,
```

Comment:
AgentVersion should not be in ServerInfo, because ServerInfo is about PMM-Server

Reply:
@BupycHuk I moved it into StatusResponse, please check if it is OK. Thank you.